debezium-connector-mysql/src/test/resources/ddl/connector_test.sql

```diff
@@ -70,4 +70,6 @@ VALUES (default, '2016-01-16', 1001, 1, 102),
        (default, '2016-01-17', 1002, 2, 105),
        (default, '2016-02-18', 1004, 3, 109),
        (default, '2016-02-19', 1002, 2, 106),
-       (default, '16-02-21', 1003, 1, 107);
\ No newline at end of file
+       (default, '16-02-21', 1003, 1, 107);
+
+CREATE DATABASE IF NOT EXISTS emptydb;
```

Comment:
sometimes, not always, tests were failing for me with "Unknown database emptydb". that fixed it for me.

Reply:
Ok 👍